pkgs/development/python-modules/h5py/default.nix

```diff
@@ -53,7 +51,6 @@ buildPythonPackage rec {
     substituteInPlace pyproject.toml \
       --replace-fail "numpy >=2.0.0rc1" "numpy"
   '';
-  pythonRelaxDeps = [ "mpi4py" ];
 
```

Comment:
Is this intentional? Commit messages don't mention it.

Reply:
It's intentional, reworded commit message.

Reply:
Oops this breaks python312Packages.h5py-mpi, reverted.